src/main/java/com/polishuchenko/bookstore/repository/cartitem/CartItemRepository.java

```diff
@@ -2,8 +2,11 @@
 
 import com.polishuchenko.bookstore.model.CartItem;
 import java.util.Optional;
+import java.util.Set;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface CartItemRepository extends JpaRepository<CartItem, Long> {
     Optional<CartItem> findCartItemByIdAndShoppingCartId(Long cartItemId, Long shoppingCartId);
```

Comment:
You need both id and shopping cart id for search?

Reply:
Agree, is it possible to search just for cartItemId, as far is I understand one cartItem is related to exactly one Shopping cart